include/pybind11/pytypes.h

```diff
@@ -501,7 +501,7 @@ struct error_fetch_and_normalize {
                             "active exception.");
         }
         const char *exc_type_name_norm = detail::obj_class_name(m_type.ptr());
-        if (exc_type_name_orig == nullptr) {
+        if (exc_type_name_norm == nullptr) {
```

Comment:
Thanks for catching this one! Sorry that's a mistake of mine that slipped in with #1895.